flink/src/test/java/org/apache/sedona/flink/ConstructorTest.java

```diff
@@ -30,6 +30,17 @@
 import static org.apache.flink.table.api.Expressions.call;
 import static org.junit.Assert.assertEquals;
 
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.common.typeinfo.BasicTypeInfo;
+import org.apache.flink.api.common.typeinfo.PrimitiveArrayTypeInfo;
+import org.apache.flink.api.java.typeutils.RowTypeInfo;
+
+import org.apache.flink.streaming.api.datastream.DataStream;
+
```

Comment:
Please remove unused spaces.

Reply:
Should be fixed with [3def393](https://github.com/apache/incubator-sedona/pull/628/commits/3def393bbcef98ba19651814a60a362e797a4b36).